index.js

```diff
@@ -9,10 +9,11 @@ var MemoizedConnect = async.memoize(function (alias, callback) {
   MongoClient.connect.apply(MongoClient.connect, configs[alias].concat([callback]));
 });
 
-
 var getDb = module.exports = function(alias, callback) {
   //directly using a connection string as alias.
-  if ( !(alias in configs) && typeof alias == 'string' && alias.indexOf('mongodb://') === 0 ) {
+  var isMongoUrl = alias.indexOf('mongodb://') === 0 || alias.indexOf('mongodb+srv://') === 0
```

Comment:
`alias` is an optional argument. This change is breaking some of our usages where we call `getDb()` with only a callback argument (i.e. expecting the `"default"` entry as you can see at line 22 below).
```
/var/lib/jenkins/workspace/auth0-worker-lib_PR-117/node_modules/mongo-getdb/index.js:14
  var isMongoUrl = alias.indexOf('mongodb://') === 0 || alias.indexOf('mongodb+srv://') === 0
                         ^

TypeError: alias.indexOf is not a function
```